zeppelin-web/src/app/notebook/notebook-actionBar.html

```diff
@@ -66,6 +66,23 @@
               tooltip-placement="bottom" tooltip="Export this note">
         <i class="fa fa-download"></i>
       </button>
+
+      <button type="button"
+              class="btn btn-primary btn-xs"
+              ng-if="ticket.principal && ticket.principal !== 'anonymous'"
+              ng-hide="viewOnly || note.config.personalizedMode !== 'true'"
+              ng-click="toggleNotePersonalizedMode()"
+              tooltip-placement="bottom" tooltip="Personal mode {{isOwner ? '' : '(owner can change)'}}">
+        <i class="fa fa-user"></i>
+      </button>
+      <button type="button"
+              class="btn btn-default btn-xs"
+              ng-if="ticket.principal && ticket.principal !== 'anonymous'"
+              ng-hide="viewOnly || note.config.personalizedMode === 'true'"
+              ng-click="toggleNotePersonalizedMode()"
+              tooltip-placement="bottom" tooltip="Shared mode {{isOwner ? '' : '(owner can change)'}}">
```

Comment:
it's nit. but how about "Collaboration mode" instead of "Shared mode" ? 
"Shared mode" could confuse user with interpreter "shared" mode.